### PR TITLE
Charts enhancement

### PR DESCRIPTION
**Pull request by hug33k**
This PR add two enhancements for charts :

- A tooltip button is now displayed to explain how to use filters in charts
- Total cost in Pie Chart is now updated when user select items

### CODE REVIEW

File: src/styles/style.css [hunk 1]
```diff
@@ -735,3 +735,14 @@ h4.no-data {
...
+.wrapper .popover-trigger i.btn {
+    font-size: 1.45em;
+}
+
+.wrapper .popover-trigger i.btn:active {
+    background: #ffffff;
+    color: #4885ed;
+    border-color: #f1f1f1;
+    box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.1);
+}
```
Comment (Lolincolc) [inline src/styles/style.css]: You forget to add a new line

**Comment by hug33k** [comment]
@simonmeyerrr Done

File: src/components/aws/costBreakdown/ChartComponent.js [hunk 2]
```diff
@@ -180,6 +181,12 @@ export class Header extends Component {
       </div>
     ) : null);
 
+    const tooltip = ((this.props.type === "pie" || this.props.type === "bar") ? (
+      <div className="inline-block">
+        <Tooltip icon={<i className="fa fa-info-circle btn btn-default"/>} tooltip="To hide an item, click on it. Double-click to select only one item." placement="left"/>
```
Comment (alexis-ld) [inline src/components/aws/costBreakdown/ChartComponent.js]: Not clear if you are talking about the item on the chart or on the legend. Please change it to something like : 
"To hide/display a data series please click on its name in the legend. Double-click will display only this item"

File: src/components/aws/tags/ChartComponent.js [hunk 3]
```diff
@@ -54,6 +55,12 @@ class Header extends Component {
       loading = (!this.props.values || !this.props.values.status ? (<Spinner className="spinner clearfix" name='circle'/>) : null);
     }
 
+    const tooltip = (
```
Comment (alexis-ld) [inline src/components/aws/tags/ChartComponent.js]: Why only the tooltip and not the chart title update on the tag view ?

**Pull request merged**